From a0ff12558dde6758933c545e5ec8e38b606820fc Mon Sep 17 00:00:00 2001
From: Seth Foster <seth@opentrons.com>
Date: Tue, 27 May 2025 11:05:34 -0400
Subject: [PATCH] build with numpy 1.21.6

Alter the exact numpy build requirement to match what we have locally, since we will
not be allowing pep517 automatic build environment preparation.

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 05490d5060..a2eae0d5a9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -7,10 +7,10 @@ requires = [
     "Cython>=0.29.13",  # Note: sync with setup.py
     "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
     "numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system!='AIX'",
+    "numpy==1.21.6; python_version>='3.8' and platform_system!='AIX'",
     "numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'",
     "numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system=='AIX'",
+    "numpy==1.21.6; python_version>='3.8' and platform_system=='AIX'",
 ]
 
 [tool.black]
-- 
2.47.1

